api-first-approach/index.asciidoc

```diff
@@ -0,0 +1,90 @@
+= API first approach with OpenAPI generator
+====
```

Comment:
Please add tags -> see docs on how to do it

Reply:
I have added tags, but where are the tags used? They are not added in the tutorial if I see that right